pkg/logs/config/integration_config.go

```diff
@@ -62,6 +63,13 @@ type LogsConfig struct {
 	// could have been unidirectional but the tailer could not close it in this case.
 	Channel chan *ChannelMessage
 
+	// ChannelTags are the tags attached to messages on Channel; unlike Tags this can be
+	// modified at runtime (as long as ChannelTagsMutex is held).
+	ChannelTags []string
+
+	// ChannelTagsMutex guards ChannelTags.
+	ChannelTagsMutex sync.Mutex
```

Comment:
Should this be an RWMutex?

Reply:
It won't be read concurrently, so I don't think the overhead is worthwhile.